javascript/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@dzcode-io/kuliya",
+  "version": "0.1.0",
+  "description": "Algeria's college hierarchy dataset as javascript package",
+  "main": "dist/index.js",
+  "scripts": {
+    "postinstall": "node src/_config/pre.js",
+    "data": "yarn build && node dist/_config/prepare.js",
+    "build": "tsc",
+    "build:watch": "yarn build --watch",
+    "test": "yarn build && jest src",
+    "eslint:check": "eslint \"**/*.*\" --report-unused-disable-directives",
+    "eslint:fix": "eslint \"**/*.*\" --quiet --fix",
+    "prettier:check": "prettier \"**/*.*\" --check --ignore-path ./eslint.prettierignore",
+    "prettier:fix": "prettier \"**/*.*\" --write --ignore-path ./eslint.prettierignore",
+    "lint:check": "yarn eslint:check && yarn prettier:check",
+    "lint:fix": "yarn eslint:fix && yarn prettier:fix"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged --relative"
+    }
+  },
+  "lint-staged": {
+    "*": [
+      "eslint --quiet --fix --ignore-path .eslintignore",
+      "prettier --write --ignore-path ./eslint.prettierignore"
+    ]
+  },
+  "license": "MIT",
+  "keywords": [
+    "algeria",
+    "university",
+    "college",
+    "tree",
+    "javascript",
+    "typescript",
+    "types"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "repository": "https://github.com/dzcode-io/kuliya.git"
+  },
+  "author": {
+    "email": "contact@zakiii.com",
+    "name": "Zakaria Mansouri",
+    "url": "https://www.zakiii.com"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.6",
```

Comment:
why using Babel? most of ES6 features are supported now in Node, also tsc should do the magic

Reply:
the reason for doing so, is to ensure that jest reads test files written in ES6 and above, why writing tests in ES6 although we have the same tests written on TS? well, we will have users who may use this package in ES6 or pure JS code, so we have to cover that